release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -283,6 +283,62 @@ For more information, see xref:../scalability_and_performance/what-huge-pages-do
 [id="ocp-4-8-monitoring"]
 === Monitoring
 
+[id="ocp-4-8-monitoring-alerting-rule-changes"]
+==== Alerting rule changes
+
+{product-title} 4.8 includes the following alerting rule changes:
+
+.Alerting rule changes
+[%collapsible]
+====
+* The `ThanosSidecarPrometheusDown` alert severity is updated from _critical_ to _warning_.
+* The `ThanosSidecarUnhealthy` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryHttpRequestQueryRangeErrorRateHigh` alert severity is updated from _critical_ to _warning_.
+* The `ThanosQueryInstantLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for instant queries.
+* The `ThanosQueryRangeLatencyHigh` critical alert is removed. This alert fired if Thanos Querier had a high latency for range queries.
+* For all Thanos Querier alerts, the `for` duration is increased to 1 hour.
+* For all Thanos sidecar alerts, the `for` duration is increased to 1 hour.
+
+[NOTE]
+=====
+Red Hat does not guarantee backward compatibility for metrics, recording rules, or alerting rules.
+=====
+====
+
+[id="ocp-4-8-monitoring-stack-dependency-version-updates"]
+==== Version updates to monitoring stack components and dependencies
+
+{product-title} 4.8 includes version updates to the following monitoring stack components and dependencies:
+
+* The Prometheus Operator is now on version 0.48.1.
+* Prometheus is now on version 2.26.0.
+* The `node-exporter` agent is now on version 1.1.2.
+* Thanos is now on version 0.20.2.
+* Grafana is now on version 7.5.5.
+
+[id="ocp-4-8-monitoring-kube-state-metrics-2.0.0-upgrade"]
```

Comment:
kube-state-metrics is paralleled components with Prometheus, node-exporter, Thanos and Grafana

Reply:
> kube-state-metrics is paralleled components with Prometheus, node-exporter, Thanos and Grafana

@simonpasquier Do you recommend that the above is mentioned in this note regarding kube-state-metrics?